### DISCUSSION TITLE
Icon 组件

### DISCUSSION THREAD

**Discussion by irisSong** [jdf2e/nutui-react#23] [General]
className默认值为空，52，53不需要对className在判断

**Comment by richard1015** [comment]
- 功能、基本交互测试
   - [ ] demo 基本样式有问题，需要采用自定义config进行分类，具体可以参考nutui vue
   - [ ] ...
- 文档完整性验证
   - [ ] import { Icon } from '@nutui/nutui-react';
   - [ ] ...
- props 所有属性测试
   - [ ] ...
   - [ ] ...
- props 异步修改组件是否动态更新
   - [ ] ...
   - [ ] ...
- emits 所有事件测试
   - [ ] ...
   - [ ] ...
- npm run build 子组件是否被构建
   - [ ] ...
   - [ ] ...
- 其它
   - [ ] ...
   - [ ] ...
